packages/verification-pool-api/src/config.ts

```diff
@@ -35,6 +38,38 @@ export function getTwilioClient() {
   return twilioClient
 }
 
+export function getNexmoClient() {
+  if (nexmoClient == null) {
+    nexmoClient = new Nexmo({
+      apiKey: functionConfig.shared['nexmo-key'],
+      apiSecret: functionConfig.shared['nexmo-secret'],
+    })
+  }
+  return nexmoClient
+}
+
+export async function sendSmsWithNexmo(countryCode: string, phoneNumber: string, message: string) {
```

Comment:
Nit: Config.ts seems like the wrong place for this

Reply:
Given the relative urgency of this, do you think we can do this in a future PR?

Reply:
ok